WcaOnRails/app/views/relations/relation.html.erb

```diff
@@ -0,0 +1,49 @@
+<% provide(:title, t('.title')) %>
+
+<div class="container text-center">
+  <h3><%= t '.find_relations' %></h3>
+
+  <%= form_tag relation_url, method: :get, class: "form-inline relations-form" do %>
```

Comment:
Ok, if you tried it already and didn't like it, then this must be the way to go!